applications/callflow/src/cf_endpoint.erl

```diff
@@ -116,6 +118,14 @@ maybe_have_endpoint(JObj, EndpointId, AccountDb) ->
             has_endpoint(JObj, EndpointId, AccountDb, EndpointType)
     end.
 
+-spec maybe_map_endpoint_type(api_binary()) -> api_binary().
+maybe_map_endpoint_type('undefined') -> 'undefined';
+maybe_map_endpoint_type(EndpointType) ->
```

Comment:
@jamesaimonetti no particular reason. this way allows to move it to whapps_account_config/whapps_config if there is a request to support it by cluster / account_id.